### PR TITLE
added some text to the index.html file

### PR DESCRIPTION
**Pull request by fmontufar**
@wykhuh: Please check my code. :)

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -1,6 +1,29 @@
 {% extends "layout.html" %}
 {% block content %}
 
-home
+<div>
+We are citizens who are interested in helping JPL identify features of interest in images collected by the Mars rover.
```
Comment (wykhuh) [inline templates/index.html]: missing `<p>`

File: templates/index.html [hunk 2]
```diff
@@ -1,6 +1,29 @@
...
+
+The features JPL needs us to identify are listed below.
+
+<li>Layered rocks:<br>
+http://mars.jpl.nasa.gov/msl-raw-images/msss/00753/mcam/0753MR0032340000403661E01_DXXX.jpg</li>
```
Comment (wykhuh) [inline templates/index.html]: missing `<a href>`

File: templates/index.html [hunk 3]
```diff
@@ -1,6 +1,29 @@
 {% extends "layout.html" %}
 {% block content %}
 
-home
+<div>
+We are citizens who are interested in helping JPL identify features of interest in images collected by the Mars rover.
+
+The features JPL needs us to identify are listed below.
+
```
Comment (wykhuh) [inline templates/index.html]: missing `<ul>`

**Comment by benmont99** [comment]
@wykhuh : Here is the revised index.html

**Comment by wykhuh** [comment]
@benmont99 :+1:

**Pull request merged**